.ncurc.json

```diff
@@ -1,5 +1,5 @@
 {
   "upgrade": true,
   "deep": true,
-  "reject": ["nanoid"]
+  "reject": ["uid"]
```

Comment:
we dont need this anymore - it was only here to prevent `ncu` from updating nanoid automatically

```suggestion
```